app/digests/events.py

```diff
@@ -0,0 +1,6 @@
+from elife_bus_sdk.events import Event
+
+
+class DigestEvent(Event):
```

Comment:
seems good enough to define the `DigestEvent` here rather than in the library, since the base library doesn't have to know all the possible types from all the services then